packages/@lwc/engine-core/src/framework/template.ts

```diff
@@ -111,7 +113,27 @@ function validateLightDomTemplate(template: Template, vm: VM) {
 const enum FragmentCache {
     HAS_SCOPED_STYLE = 1 << 0,
     SHADOW_MODE_SYNTHETIC = 1 << 1,
-    HAS_LEGACY_SCOPE_TOKEN = 1 << 2,
+}
+
+// This should be a no-op outside of LWC's Karma tests, where it's not needed
+let registerFragmentCache: (fragmentCache: any) => void = noop;
+
+// Only used in LWC's Karma tests
+if (process.env.NODE_ENV === 'test-karma-lwc') {
+    // Keep track of fragmentCaches, so we can clear them in LWC's Karma tests
+    const fragmentCaches: any[] = [];
+    registerFragmentCache = (fragmentCache: any) => {
+        fragmentCaches.push(fragmentCache);
+    };
+
+    // @ts-ignore
+    window.__lwcResetFragmentCaches = () => {
```

Comment:
```suggestion
    (window as any).__lwcResetFragmentCaches = () => {
```
`@ts-ignore` is gross. So is `any`, but it's less gross, and it seems overkill to do it the right way.
```ts
declare global {
    interface Window {
        __lwcResetFragmentCaches?: unknown;
    }
}
```